doc/rules/alias/modernize_strpos.rst

```diff
@@ -0,0 +1,61 @@
+=========================
+Rule ``modernize_strpos``
+=========================
+
+Replace ``strpos()`` expressions with ``str_starts_with()`` or
+``str_contains()`` if possible.
+
+.. warning:: Using this rule is risky.
+
+   Risky if the ``strpos`` function is overridden.
```

Comment:
or if the `str_starts_with` or `str_contains` functions are.

Reply:
:+1:  added